### ISSUE TITLE
processor.starlark add constant definition in processor configuration

### ISSUE THREAD

**Issue by mirkocomparetti-synesis** [influxdata/telegraf#8768]
## Feature Request

The starlark plugin can run scripts on the data; it is not possible to define some constants on the telegraf processor configuration to be passed along to the `apply` method and be used in the script.

### Proposal:

Add the possibility to define constants in the configuration of the processor to be used in the script, as constant values.

### Current behavior:

Imagine the floowing configuration
```
[[processors.starlark]]
  script = "/path/to/myscript.star"
```
and the following script (taken from the examples, as a reference)
```
def apply(metric):
    if metric.fields["mycustomfield"] >= 10:
      return metric
    else:
      return None
```
now, if in another case we need to filter numbers with a different value (different from 10) we need to create another script with the same method and different values.

### Desired behavior:

Using the same example above as a reference, define constants (something) like
```
[[processors.starlark]]
  somecustomstr = "mycustomfield"
  somecustomnum = 10
  script = "/path/to/myscript.star"
```
to use them in the script in this way
```
def apply(metric):
    if metric.fields[somecustomstr] >= somecustomnum:
      return metric
    else:
      return None
```
The syntax si just there to clarify, anything different will do as well.

### Use case: 

In some case the same script needs to be used in different starlark processors with different constant values. Having such a feature, will make it possible to define a _library_ of scripts to be used whenever needed, changing the constant values in the definition of the processor.

**Comment by essobedo** [comment]
The corresponding PR https://github.com/influxdata/telegraf/pull/8772

**Comment by sjwang90** [comment]
Closed in #8772
